src/components/progressBar/ProgressBar.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {SafeAreaView, StatusBar, Text, View} from 'react-native';
+import {styles} from './progressBar.style';
+
+export const ProgressBar = ({value = '5%', max = '100'}) => {
```

Comment:
use only numbers. don't use string with percentige

Reply:
added the changes for v.2